### PR TITLE
Update wis2node.adoc focusing on Registration, decomissioning of nodes & datasets, node performance metrics

### PR DESCRIPTION
**Pull request by klsheets**
For Review: Updates to 2.2.1 Practices and Procedures, 2.2.2 Publishing data, metadata, discovery metadata and notification messages, and 2.2.3 Performance metrics.

All sections likely will need formatting reviews.  Some things I noted while working are  when do we use WIS vs WIS2 - specifically about nodes, but I think there are few other places where they are used interchangeably.

For performance metrics, the template asked specifically for WIS nodes and not global services.  We focused more on the performance metrics for global services, I can add these if desired, but I stuck to nodes and drew more on what happens currently for GISCs and GISC watch for what I wrote.

### CODE REVIEW

**Comment by 6a6d74** [comment]
Hi @klsheets ... thanks for committing these changes :)

Comments below:

General: you often refer to "the WMO" (e.g., lines, 9, 12). Better to refer to "WMO Secretariat" I think?

Line 9 (describing `centre-id`): the first token (`ab`) is based on an IANA Top Level Domain not ISO3166 2-letter code. Also, the third token (`nodename`) is optional. For example, a Member may only have a single WIS2 Node - in which case there's nothing to distinguish between at the organisation level.

Line 12 (describing decommissioning): Your proposal works for a NC. But is there an additional step for DCPC in contacting the "sponsor" (i.e., Regional Association or WMO Programme)? To be considered.

In case for both NC and DCPC I think it's worth stating that the operator shall ensure that obligations relating to data sharing within WIS continue to be met after the WIS Node is decommissioned, for example, by migrating these data sharing obligations to another WIS Node. 

(in the case of DCPC, the other WIS Node may or may not be operated by the same Member, I guess)

Line 26 (discussing metadata): Data Publishers may work with their affiliated GISC to develop good metadata, but I thought that the GDC verified the metadata? Based on that verification, the GDC will publish a report indicating errors and/or potential improvements (based on discovery metadata KPIs). The GISC should work with the data publisher to remedy issues.

Also, I thought that the topic "recognition" was automated?
1. GDC verifies that the topic is valid.
2. By way of subscribing to notification messages about discovery metadata, the GBs automatically parse the (local) message broker address and topics to which it should subscribe
3. GCs have a blanket subscription to at least two GBs for everything on data/core and metadata/core - so they don't need to know about these updates. 

@golfvert, @tomkralidis is this correct? ^^

In addition, the data publisher's affiliated GISC should do some systematic review of what's being published to make sure everything is working OK.  

Section "Publication and topic selection": Should add something about using the "experimental" topic for getting started with pre-operational data exchange in test mode ... `experimental` sits under domain (level 8), e.g. `...weather/experimental`. Data publishers can can extend the experimental branch with sub-topics as they deem appropriate. Data consumers must not assume that experimental topics will be durable (i.e., they may change or be removed).

Line 97: spelling ... methode >> method

Line 103: spelling ... q >> a

Line 103: I don't think there are implementations of option 3. Perhaps better to say that option 3 involves bilateral arrangement with another centre to publish metadata on their behalf, such as via their affiliated GISC.
... we could also note that WIS2 Portals may offer metadata publication services in future.

Line 136: spelling ... GISCC >> GISC

**Comment by klsheets** [comment]
Thank you @6a6d74 for the quick review and guidance,

WMO Secretariat makes sense to me, I was following some of the diagrams when I went with WMO, but in my own self edit I was flipping between WMO and Secretariat and changes all to WMO, so adding Secretariat makes sense to me as well.

I'll make that change and the typo/grammar connections you pointed out.

The things you point out regarding the the GDC and authenticating metadata and the role of GISCs are the items I struggled most to document when drafting these edits.  I was largely following the workflows diagrams we reviewed in November, provided by Remy and Annexes to the meeting report.  Which brings up and additional question - should those diagrams be linked or included as an Annex in the guide?

**Comment by 6a6d74** [comment]
@klsheets - thanks for updating. 

I think we'll need another round of review for how the _process_ will work before we can write this in the documentation. I've raised a new issue (#69) to capture this action.

@tomkralidis - I recommend merging Kari's changes into the main branch as a baseline for further updates in the coming week or so.

**Comment by tomkralidis** [comment]
Suggest to merge once #67 is approved/merged, if possible, pending some updates from @golfvert

**Comment by kurt-hectic** [comment]
@klsheets . The broker endpoints are mandatory, I thus suggest the text to read "*must* provide". You may also consider to be more explicit that broker hostname, port(s) and supported protocols *must* be supplied *to the WMO Secretariat* as part of the Registration of a Node (in addition to other elements).
It could also be considered to link to a WMO community webpage which provides details about the registration process.

**Comment by klsheets** [comment]
@kurt-hectic @golfvert @6a6d74 I thought there was sensitivity to wording such as "must" in the guide. I know during the ET2AT meeting Fred, as a former CBS president, said we could use shall.  Happy to word it as strongly as is permissible.

**Comment by kurt-hectic** [comment]
We must indeed write shall. My point was that this information is mandatory, not optional.


________________________________

File: guide/sections/part2/wis2node.adoc [hunk 1]
```diff
@@ -4,24 +4,33 @@
...
...e ab is the the IANA Top Level Domain code, domain is the organization's domain name for the main website, and nodename (optional) is descriptive of the purpose of the WIS node (for example a node for a type of data or programme of the data e.g. climate, aviation, etc.) when the organization is hosting multiple WIS nodes. A sample centre-ID is ab-myorg-climate.  In addition to the centre-ID the registering organization should also provide the broker endpoints.  Once centre-ID and broker endpoints are provided and entered into the WIS2 node database, the WMO Secretariat will contact the GISC for the organization's country and request the GISC to verify the correctness of the provided information.  The GISC will request the centre produce a test message with associated files for download.  The GISC should use a test MQTT client to verify the notification message is correct and the download links are functional.  The GISC notifies the Centre of the results of the checks and if all are good, requests the Centre to provide metadata when the dataset is ready.  When metadata is proivided the GISC informs the WMO Secretariat that the new centre-ID is ready and requests it be added to WIS2.
```
Comment (amilan17) [inline guide/sections/part2/wis2node.adoc:9]: "Where ab is the the IANA Top Level Domain code, domain is the organization's domain name for the main website, and nodename (optional) is descriptive of the purpose of the WIS node (for example a node for a type of data or programme of the data e.g. climate, aviation, etc.) when the organization is hosting multiple WIS nodes."

Information about the centre-id rules will be in an appendix of the Manual for Topic Hierarchy.... Maybe add a reference to that here instead?

File: guide/sections/part2/wis2node.adoc [hunk 2]
```diff
@@ -4,24 +4,33 @@
...
 
 ===== Registration and removal of a Dataset
 
-This section describes the process used to register a Dataset so that it may be discovered and shared within WIS2. In cases where a WIS Centre no longer wishes to share a Dataset via WIS, it must be removed as per the procedure described here.
+All data shared in WIS2 must include metadata compliant with the metadata requriements described below.  Data providers must be approved by their PR (or designate such as the member's WIS Focal Point) to ensure quality metadata will be provied along with the data.
 
-TODO: to be completed
+An organization ready to publish a new dataset should contact the WMO Secretariat with the approval of their PR (or designate) to The WMO Secretariat.  The WMO Secretariat will contact the GISC for the organizaiton with the new (meta)data.  The GISC will work with a Global Data Catalogue to verify the metadata.  the GDC will publish a report indicating errors and/or potential improvements (based on discovery metadata KPIs). The GISC should work with the data publisher to remedy issues and incporate suggestions for improvement.
```
Comment (amilan17) [inline guide/sections/part2/wis2node.adoc:20]: Is this workflow for each new dataset? I would expect that once a WIS2. node has been approved that this metadata and data workflow would occur without a lot of oversight and regular monitoring should be sufficient.

"An organization ready to publish a new dataset should contact the WMO Secretariat with the approval of their PR (or designate) to The WMO Secretariat. The WMO Secretariat will contact the GISC for the organizaiton with the new (meta)data. The GISC will work with a Global Data Catalogue to verify the metadata."

File: guide/sections/part2/wis2node.adoc [hunk 3]
```diff
@@ -83,25 +92,56 @@ Third, the Open Geospatial Consortium (OGC) have developed a suite of APIs (call
...
+
+    Publish initial metadata.  
+    
+    Publish update metadata.  
+    
+    Data update notification: normal notification message with property.cache=false  
+
+Use of the "experimental" topic  
+
+    The "experimental" topic is necessary for the WIS2 pre-operational phase and future pre-operational data exchange in test mode.
```
Comment (amilan17) [inline guide/sections/part2/wis2node.adoc:119]: experimental topic may also be necessary when the sub-topics have not been approved yet

**Pull request merged**